internal/appsec/emitter/graphqlsec/execution.go

```diff
@@ -0,0 +1,86 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+// Package graphql is the GraphQL instrumentation API and contract for AppSec
+// defining an abstract run-time representation of AppSec middleware. GraphQL
+// integrations must use this package to enable AppSec features for GraphQL,
+// which listens to this package's operation events.
+package graphqlsec
+
+import (
+	"context"
+	"reflect"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/trace"
+)
+
+type Execution struct {
```

Comment:
I'd suggest to align with the naming convention we used in httpsec and grpcsec, and adding the `Operation` suffix on those new graphql  operations too.

A global renaming can be discussed and done on a separate PR.

Reply:
Fixed in 1affe2e98696ba6c06d6ff4400d378b9c389ecc8